rocky/rocky/views/mixins.py

```diff
@@ -510,6 +511,16 @@ def get_ooi_properties(self, ooi: OOI):
 
         props.pop("scan_profile")
         props.pop("primary_key")
+        if "user_id" in props and props["user_id"]:
+            user = get_user_model().objects.get(id=props["user_id"])
+            if user.is_active:
+                props["user_id"] = str(user)
+            else:
+                name = str(user)
+                props["user_id"] = "".join([c + "\u0336" if i < len(name) - 1 else c for i, c in enumerate(name)])
```

Comment:
Lets do the strikestrough in the html frontend.

Reply:
This will required to pass a additional variable to the frontend, the current solution does not.

Reply:
Done in b8b1465baadaba7b233caa712c697336aca6e4e0.